### PR TITLE
PFHub Upload: PRISMS_PF_3a_cubic

### PR DESCRIPTION
**Pull request by staticmanapp**
This a new PFHub upload

To review:

- Wait for the tests to pass.

- Go to http://random-cat-825.surge.sh/simulations/display/?sim=PRISMS_PF_3a_cubic

| Field     | Content                                                                                         |
| --------- | ----------------------------------------------------------------------------------------------- |
| metadata  | [object Object]                                                                                 |
| benchmark | [object Object]                                                                                 |
| data      | [object Object],[object Object],[object Object],[object Object],[object Object],[object Object] |
| date      | 1535054177                                                                                      |

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/825)

### CODE REVIEW

**Comment by wd15** [comment]
@stvdwtt, this is ready for you to review.
 
 - http://random-cat-825.surge.sh/simulations/display/?sim=PRISMS_PF_3a_cubic
 - http://random-cat-825.surge.sh/simulations/3a.1/

There is an issue with the zero level set. The points need to be ordered correctly.

File: _data/simulations/PRISMS_PF_3a_cubic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    type: line
+    transform:
+      - type: formula
+        expr: datum.time
+        as: x
+      - type: formula
+        expr: datum.solid_fraction
+        as: 'y'
+  - name: phase_field_1500
+    url: 'https://f000.backblazeb2.com/file/pfhub-sjd/phase_field_1500.csv'
```
Comment (wd15) [inline _data/simulations/PRISMS_PF_3a_cubic/meta.yaml]: Need to fix this data so the points are ordered

**Comment by stvdwtt** [comment]
@wd15 I've fixed the ordering for the contour plot. In #827, I made some minor edits to the metadata for the simulation. Once that's merged in, I think this will be good to go.

**Comment by wd15** [comment]
@stvdwtt, shall I close this pull request?

**Comment by wd15** [comment]
@stvdwtt, sorry, ignore that last comment. Your patch is a pull request into this pull request. Confusing!

**Comment by stvdwtt** [comment]
@wd15 Ah -- got it. Correct, the patch is an edit to this PR. Is there a more straightforward way to do that?

**Comment by wd15** [comment]
@stvdwtt, ok merging this now. I think you did the right thing on the pull request. I was just confused.

**Comment by stvdwtt** [comment]
👍

**Pull request merged**